account_financial_report/wizard/general_ledger_wizard_view.xml

```diff
@@ -95,9 +93,7 @@
                         </page>
                     </notebook>
                 </div>
-                <div
-                    attrs="{'invisible': [('not_only_one_unaffected_earnings_account', '=', False)]}"
-                >
+                <div invisible="not_only_one_unaffected_earnings_account == False">
```

Comment:
```suggestion
                <div invisible="not not_only_one_unaffected_earnings_account">
```

Although I think the variable name should be reversed for not having these ugly expressions.

There are more occurrences of the same kind, so this apply to all.

Reply:
Done